client/watcher/watcher.go

```diff
@@ -74,6 +74,7 @@ func (w *Watcher) Watch() {
 			w.mu.Lock()
 			select {
 			case event := <-w.watcher.Events:
+				printer.LogErrf("[dcdr] event log: %s", event.String())
```

Comment:
should this be an error? using `printer.Info` might make more sense here if not

Reply:
oh fair